src/Compilers/VisualBasic/Portable/Symbols/Source/SynthesizedMyGroupCollectionPropertyAccessorSymbol.vb

```diff
@@ -169,10 +170,12 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
             '    return <backingField>
             ' End Get
 
-            Return "Get" & vbCrLf &
```

Comment:
I'm referring to the indentation of the body of the Getter being indented relative to the `Get` and `End Get` statements.